packages/api/controllers/application.js

```diff
@@ -70,16 +73,45 @@ module.exports.put = async (req, res, next) => {
 
 module.exports.deploy = async (req, res, next) => {
   if (!validatePropertyName(req, next)) return;
-  const userId = getUserUUID(req);
+  const userId = await extractUserId(req);
   const name = getNameRequest(req);
   const identifier = getIdentifier(name);
   const nextRef = getRefRequest(req);
-  const ref = getRefRequest(req);
+  const ref = "";
```

Comment:
if this is not being used anywhere we can remove this

Reply:
no deepesh, we'll update it once we receive the ack, we are using this in deployment service